build/generate-docs.ts

```diff
@@ -144,9 +144,10 @@ function requiresToMarkdown(requires: any[]): string {
  * @param type - the type of the property
  * @returns the markdown link string
  */
-function typeToMakrdownLink(type: string): string {
+function typeToMarkdownLink(type: string): string {
     switch (type.toLocaleLowerCase()) {
         case '*':
+        case 'variableanchoroffsetcollection':
```

Comment:
Can you check that the build fails if you remove this line? i.e. remove, commit, bring back, commit and see that the first run fails and the second passes.

Reply:
<img width="976" alt="image" src="https://github.com/maplibre/maplibre-style-spec/assets/649392/b792ece1-121d-4ecc-90a8-ff18538c9619">

Reply:
Great! thanks!